app/src/main/webapp/app/shared/gridresize.ctrl.js

```diff
@@ -0,0 +1,15 @@
+esbMessageAdminApp.controller('gridResizeController', [
```

Comment:
Looks like this is only used in the errors tab since it's the only thing that uses the grid. Until it's used elsewhere (which likely may never happen), what do you think about moving this to the error/ folder since it only exists to support that tab?

As a bonus, if you can think of a better name for this controller, go for it. It's not just a generic "grid resize controller," which to me sounds like it is an independent controller about providing resizing functionality. It's not though; it's strictly for broadcasting an "errorGridResize" event to interested consumers. Maybe `GridResizeBroadcastController`?